dummy-data/competitions.js

```diff
@@ -0,0 +1,25 @@
+const competitions = [
```

Comment:
Maybe it's good to have an id to every element of arrays to pin a key value for mapping to be possible ?

Reply:
good notice! I'll make some stiff data, but maybe in connection it will be good as another task to make id-generator? for future, just something to talk if we'll need it

Reply:
Added to competitors and competitions, doggos has pkr or register number or chip number, all of them are unique

Reply:
maybe we can use the uuid addon? there is no need to reinvent the wheel.
https://www.npmjs.com/package/uuid